core/src/test/java/org/bitcoinj/core/TxConfidenceTableTest.java

```diff
@@ -46,6 +46,7 @@ public class TxConfidenceTableTest {
     public void setup() throws Exception {
         BriefLogFormatter.init();
         Context context = new Context(UNITTEST);
+        Context.propagate(context);
```

Comment:
Why did you make this change here and nowhere else?

Reply:
Because that test needs the correct context propagated. The others succeed without, so I didn't take any steps. Would you prefer me explicitly propagating context where one is instantiated?